examples/uart/src/uart_test.py

```diff
@@ -0,0 +1,59 @@
+import serial
+import time
+
+# Configure the serial port
+SERIAL_PORT = '/dev/tty.usbmodem0011600012981'  # apollo4l serial port
```

Comment:
There is a way to find the port automatically by looking for a uuid - see https://github.com/AmbiqAI/ns-mirror/blob/4c18e561603651d922afdaa26dedb8fbddccd78d/tools/ns_utils.py#L195 for how I did it for autodeploy